istio-control/istio-discovery/values.yaml

```diff
@@ -94,4 +94,17 @@ pilot:
     # Will not define mixerCheckServer and mixerReportServer
     enabled: false
 
-  useMCP: true
\ No newline at end of file
+  useMCP: true
+
+## Mixer settings
+mixer:
```

Comment:
seems having the top level mixer config in here is odd... mixer has its own config in https://github.com/huang195/installer/tree/tunnable_batch_report/istio-telemetry

Reply:
right, if it's possible to move these to `istio-telemetry/mixer-telemetry`, that would be ideal. However, the configmap where these variables are needed is located in `istio-control/istio-discovery`. These are 2 different subcharts. Afaik, subcharts in helm don't share variable scope even with their parent chart.

Reply:
Looking at it more closely, it appears they are not even subcharts, but independent charts. Can variables be shared across charts?

Reply:
I believe this is the only way to have this work. when a user is configuring things, they will probably have one custom values.yaml that they share for all the charts so it won't be weird from a UX perspective